doc/run_benchmark.rst

```diff
@@ -191,4 +191,15 @@ mentioned above. Running an arbitrary number of warmup values may also make
 the benchmark less reliable since two runs may use a different number of warmup
 values.
 
+Specializer statistics (``pystats``)
+==================================
 
+``pyperf`` has built-in support for `specializer statistics (``pystats``) <https://docs.python.org/dev/using/configure.html#cmdoption-enable-pystats>`_.
+If running benchmarks on a CPython built with the ``--enable-pystats`` flag, pyperf will automatically collect ``pystats`` on the benchmark code by calling ``sys._stats_on`` immediately before the benchmark and calling ``sys._stats_off`` immediately after.
```

Comment:
Since you go into details, you should also mention that they start by calling ``_stats_clear()``. Moreover, you mention "only if we aren't warming up or calibrating".

Reply:
I added "Stats are not collected when running ``pyperf``'s own code or when warming up or calibrating the benchmarks."  I don't think mentioning `_stats_clear()` is necessary -- the point of that is to not include `pyperf`'s own code.